qlkube/src/watch.js

```diff
@@ -69,7 +69,11 @@ function kwatch(api, label) {
         });
       },
       err => {
-        console.error(err);
+        if (err.message === 'Not Found') {
+          setTimeout(() => kwatch(api, label), 2000);
+        } else {
+          console.error(err);
```

Comment:
Add a `console.error("No items found to watch, restarting in 2s")` before the `setTimeout`
And replace the `console.error(err)` with a  `console.error("Error when watching",err)`